org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WChosenboxListEditor.java

```diff
@@ -694,7 +698,10 @@ private class WChosenboxListAssistant extends Window implements EventListener<Ev
 
 		public WChosenboxListAssistant() {
 			super();
-			refID = MColumn.get(Env.getCtx(), gridTab.getTableName(), gridField.getColumnName()).getAD_Reference_Value_ID();
+			if (gridField.getAD_Process_ID_Of_Panel() > 0)
+				refID = MProcess.get(gridField.getAD_Process_ID_Of_Panel()).getParameter(gridField.getColumnName()).getAD_Reference_Value_ID();
+			else
+				refID = MColumn.get(Env.getCtx(), gridTab.getTableName(), gridField.getColumnName()).getAD_Reference_Value_ID();
```

Comment:
I think this is not the right fix, can you try refID = gridField.getAD_Reference_Value_ID() instead ?

Reply:
Indeed - commited